mailbox/maildir/src/test/java/org/apache/james/mailbox/maildir/FullUserMaildirMailboxManagerTest.java

```diff
@@ -35,52 +34,6 @@
     class HookTests {
     }
 
-    @Nested
-    class BasicFeaturesTests extends MailboxManagerTest<StoreMailboxManager>.BasicFeaturesTests {
-        @Disabled("MAILBOX-389 Mailbox rename fails with Maildir")
-        @Test
-        protected void renameMailboxShouldChangeTheMailboxPathOfAMailbox() {
-        }
-
-        @Disabled("MAILBOX-389 Mailbox rename fails with Maildir")
-        @Test
-        protected void renameMailboxByIdShouldChangeTheMailboxPathOfAMailbox() {
-        }
-
-        @Disabled("MAILBOX-389 Mailbox rename fails with Maildir")
-        @Test
-        protected void user1ShouldBeAbleToDeleteSubmailboxByid() {
-        }
-
-        @Disabled("MAILBOX-389 Mailbox rename fails with Maildir")
-        @Test
-        protected void user1ShouldBeAbleToDeleteInboxById() {
-        }
-
-        @Disabled("JAMES-2993 mailboxId support for Maildir is partial")
-        @Test
-        protected void getMailboxByIdShouldReturnMailboxWhenBelongingToUser() {
-        }
-    }
-
-    @Nested
-    class MailboxNameLimitTests extends MailboxManagerTest<StoreMailboxManager>.MailboxNameLimitTests {
-        @Disabled("MAILBOX-389 Mailbox rename fails with Maildir")
```

Comment:
Cool! Don't forget to notify that you have been solving these maildir issues in the related ticket.